functions/src/index.ts

```diff
@@ -99,3 +99,101 @@ export const scheduledDeleteOfDocumentsSetToBeDeleted = region('us-central1')
         }
         return null
     })
+
+export const getOwnersDataByUIDs = https.onCall(async (data, context) => {
```

Comment:
Her trur eg me har eit sikkerhets-issue. Bruker A kan kalle denne funksjonen med ei `ownersList` med vilkårlige bruker-UIDs og dermed få ut e-postadresser.

Eg trur det er lurare å heller kalle funskjonen f.eks. `getOwnersDataByBoardId`, altså at den tar tavle-ID, og så kan du sjekke at den brukaren som kallar funksjonen er ein av eigarane av tavla.

Reply:
Vil gjerne ha litt feedback på dette ja, for jeg var ikke helt sikker på hvordan dette bør håndteres. Var originalt gjort sånn at du måtte være eier for å få hentet dataen (og jeg lurer på om jeg kanskje kan legge det til igjen), men det ble problemer når man skal legge til folk i tavlen:

- Problem 1: Per nå får man feedback på om man har lagt til noen eller skrevet inn en epost som ikke finnes. Dette er bra for feedback, men lar deg finne ut hvilke e-postadresser som har bruker på tavla. Hvis man ikke gir feedback på det kan man heller ikke ha en 'venter på svar' status, for da ville man visst at det funket hvis man fikk opp der, eller så ville listen potensielt fylles med ikke-eksisterende brukere og man venter kanskje på en bruker som ikke finnes. Hvis man ikke legger til i noe liste før den andre har godkjent kan det også være forvirrende mtp om man har gjort det riktig og sånt. 
- Problem 2: Men nåværende implementasjon finner jeg ingen måte å legge til sikkerhetsjekk på brukere som skal søke opp e-postadresser. Det du skriver med at man kan sjekke om de er i owners eller requests på minst én tavle vil jo gjøre at man aldri kan bli med i den føreste tavlen siden man ikke oppfyller kravene, vil det ikke?

Kan gjerne ringes over Slack eller no hvis du har tid for å diskutere hvordan sikkerhet bør håndteres her,  (kan kanksje ta litt tid å ta alt skriftlig). Per nå er det jo heller ingen sikkerhet på brukere generelt i form av at hvem som helst kan lage en bruker med hvilken som helt e-post adresse (den må jo ikke bekreftes).